crates/bevy_ecs/examples/resources.rs

```diff
@@ -0,0 +1,42 @@
+use bevy_ecs::prelude::*;
+use rand::Rng;
+use std::ops::Deref;
+
+// This is our resource
+#[derive(Debug)]
+struct Counter {
+    pub value: i32,
+}
+
+fn main() {
+    // Create a world
+    let mut world = World::new();
+
+    // Add the counter resource
+    world.insert_resource(Counter { value: 0 });
+
+    // Create a schedule and a stage
+    let mut schedule = Schedule::default();
+    let mut update = SystemStage::parallel();
+
+    // Add systems to increase the counter and to print out the current value
+    update.add_system(increase_counter.system().label("increase"));
+    update.add_system(print_counter.system().after("increase"));
+    schedule.add_stage("update", update);
+
+    for iteration in 1..=10 {
+        println!("Simulating frame {}/10", iteration);
+        schedule.run_once(&mut world);
```

Comment:
I don't think we should use `Schedule::run_once` as it encourages people to run the schedule in a way that ignores schedule run criteria. We should use `Schedule::run` instead.